app/_data/tables/version_errors_konnect.yml

```diff
@@ -1,258 +1,297 @@
     DocumentationURL: |
       [Rate Limiting Advanced plugin](/hub/kong-inc/rate-limiting-advanced/)
   - ID: D112
     Severity: error
     Description: |
-      For the `proxy-cache-advanced` plugin, one or more of the following
-      `config` fields are set: `redis.username`, `redis.sentinel_username` but
-      Kong Gateway versions < 2.8.0.0 do not support these fields. Plugin features that
-      rely on these fields are not working as intended.
-    Resolution: Please upgrade Kong Gateway to version `2.8.0.0` or above.
+      For the `proxy-cache-advanced` plugin, one or more of the following `config`
+      fields are set: `redis.username`, `redis.sentinel_username` but Kong Gateway
+      versions < 2.8.0.0 do not support these fields. Plugin features that rely on
+      these fields are not working as intended.
+    Resolution: |
+      Please upgrade Kong Gateway to version `2.8.0.0` or above.
     DocumentationURL: |
       [Proxy Cache Advanced plugin](/hub/kong-inc/proxy-cache-advanced/)
   - ID: D113
     Severity: error
     Description: |
-      For the `canary` plugin, one or more of the following `config`
-      fields are set: `canary_by_header_name` but Kong Gateway versions < 2.8.0.0
-      do not support these fields. Plugin features that rely on these fields are not
-      working as intended.
-    Resolution: Please upgrade Kong Gateway to version `2.8.0.0` or above.
+      For the `canary` plugin, one or more of the following `config` fields are set:
+      `canary_by_header_name` but Kong Gateway versions < 2.8.0.0 do not support these
+      fields. Plugin features that rely on these fields are not working as intended.
+    Resolution: |
+      Please upgrade Kong Gateway to version `2.8.0.0` or above.
     DocumentationURL: |
       [Canary plugin](/hub/kong-inc/canary/)
   - ID: D114
     Severity: error
     Description: |
-      For the `forward-proxy` plugin, one or more of the following `config`
-      fields are set: `https_proxy_host`, `https_proxy_port` but Kong Gateway versions
-      < 2.8.0.0 do not support these fields. Plugin features that rely on these fields
-      are not working as intended.
-    Resolution: Please upgrade Kong Gateway to version `2.8.0.0` or above.
+      For the `forward-proxy` plugin, one or more of the following `config` fields are
+      set: `https_proxy_host`, `https_proxy_port` but Kong Gateway versions < 2.8.0.0
+      do not support these fields. Plugin features that rely on these fields are not
+      working as intended.
+    Resolution: |
+      Please upgrade Kong Gateway to version `2.8.0.0` or above.
     DocumentationURL: |
       [Forward Proxy plugin](/hub/kong-inc/forward-proxy/)
   - ID: D115
     Severity: warning
-    Description: N/A
-    Resolution: N/A
-    DocumentationURL:
+    Description: |
+      N/A
+    Resolution: |
+      N/A
+    DocumentationURL: |
+      [Forward Proxy plugin](/hub/kong-inc/forward-proxy/)
```

Comment:
I should probably remove the documentationURL, but we have one other `N/A` change where we include it. Not sure why...